templates/solutions/telco/5g-edge.html

```diff
@@ -0,0 +1,809 @@
+      <a href="https://www.hpe.com/psnow/doc/a50008940enw">
+        <h2>Discover an HPE-validated design with Canonical MicroCloud&nbsp;&rsaquo;</h2>
+      </a>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="p-section--shallow u-fixed-width">
+      <hr />
+      <h2>Bring data processing and AI/ML capabilities to the edge</h2>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4 col-9 col-start-medium-2 col-medium-5">
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-start-large-4 col-6 col-medium-3 col-start-medium-3">
+              <p>
+                Automate the lifecycle and reduce the management complexity of your distributed AI/ML and data applications.
+                Canonical delivers leading open source solutions packaged as software operators (<a href="https://juju.is/">Juju charms</a>)
+                to alleviate your management burdens.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="p-section--shallow">
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <div class="col-3 col-medium-2">
+              <h3 class="p-heading--5">Multi-cloud data solutions portfolio</h3>
+            </div>
+            <div class="col-6 col-medium-3">
+              <p>
+                Process data where it’s generated and benefit from Canonical’s portfolio of highly automated data solutions.
+                Our distributions for popular open source data management tools like Kafka, MongoDB, MySQL, OpenSearch, Postgres
+                and Spark enable you to automate complex operational tasks.
+              </p>
+              <hr class="p-rule--muted" />
+              <p>
+                <a href="/data">Learn more about our Data Solutions portfolio&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="p-section--shallow">
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <div class="col-3 col-medium-2">
+              <h3 class="p-heading--5">Integrated MLOps</h3>
+            </div>
+            <div class="col-6 col-medium-3">
+              <p>
+                Easily and reliably deploy your live datasets and AI models to bring AI inference to the edge thanks
+                to straightforward integration between our AI/ML and data solutions charms.
+              </p>
+              <hr class="p-rule--muted" />
+              <p>
+                <a href="https://charmed-kubeflow.io/">Discover Charmed Kubeflow&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <hr class="p-rule--highlight" />
+        <h2 class="p-text--small-caps">Trusted by leading telco providers</h2>
```

Comment:
```suggestion
        <h2 class="p-text--small-caps">See what industry players have to say</h2>
```
Is what it says in the doc, not sure which is right though

Reply:
Changed it to match the copy docs